lib/handlers/middleware.ts

```diff
@@ -40,6 +40,18 @@ export const jsonOnly: express.Handler = (req, res, next) => {
     return next();
 };
 
+/**
+ * Parse the incoming request as urlencoded FormData.
+ */
+export const formData: express.Handler = (req, res, next) => {
+    const defaultMaxBodySize = ceProps('maxBodySize', '1mb');
+    const parser = express.urlencoded({
```

Comment:
This is a little tricky I think, won't this create the parser and read settings every time the route is hit? Not sure we want that?

Reply:
Yeah I think I might just create it once and pass it around

Reply:
I'll move the formData parser to app.ts in the meantime.